README.md

```diff
@@ -1 +1,70 @@
-Interface mockup
\ No newline at end of file
+# Microscopium User Interface
+
+Prerequisites
+-------------
+
+* Node.js > v0.10.35
+
+Install Instructions
+--------------------
+
+```
+$ npm install
+$ bower install
+```
+
+Run dev version locally with
+```
+grunt
```

Comment:
`$ grunt`? For consistency with above. Additionally, you can get slightly nicer highlighting by adding "console" after the triple backticks.